docs/example-scenario/apps/fully-managed-secure-apps.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Fully managed applications built for secure access
+titleSuffix: Azure Example Scenarios
+description: Securely deploy an expense application to Azure App Service Environment
+author: fmustaf
+ms.date: 2/6/2019
+ms.topic: example-scenario
+ms.service: architecture-center
+ms.subservice: example-scenario
+ms.custom:
+    - fasttrack
+    - security
+social_image_url: /azure/architecture/example-scenario/apps/media/ilb-ase-with-architecture.png
+---
+
+# Fully managed applications built for secure access
+
+This scenario provides an overview of deploying secure applications using the [Azure App Service Environment (ASE)][intro-to-app-svc-env]. To restrict application access from the Internet, the Azure Application Gateway service and Web Application Firewall are used. We'll also cover best practices around continuous integration & continuous deployment (CI/CD) for App Service Environments using Azure DevOps.
+
+This is a commonly deployed scenario in industries such as banking and insurance where customers are conscious of platform level security in addition to application level security. We'll use an application that allows users to submit expense reports in this example.
+
+## Relevant use cases
+
+Consider this scenario for the following use cases:
+
+* Building an Azure Web App where additional security is required
+* Dedicated tenancy is needed, rather than shared tenant App Service Plans
+* Utilize Azure DevOps capabilities with the ILB ASE deployed inside a VNET
```

Comment:
Per style guide, spell out "internal Load Balancer".

Reply:
Also "internal load balancer app service environment" doesn't sound right, does this mean the ILB that is deployed inside the ASE?

Reply:
Per style guide, don't abbreviate VNET. Spell out "virtual network" for an instance and "Virtual Network" for the name of the Azure service.